cloudevents/tests/test_with_sanic.py

```diff
@@ -46,11 +46,12 @@ async def echo(request):
         lambda x: x
     )
     hs, body = m.ToRequest(event, converters.TypeBinary, lambda x: x)
+    print('README STUFF', hs, body)
```

Comment:
Can you elaborate a little more?

Reply:
Sorry! It seems like you've added print statements for debugging and other code changes that don't affect the behavior of this test (like changing import statements and other formatting).

If you want to make those formatting changes, I'd recommend doing it in a separate PR and add the appropriate linters along with it, otherwise it just makes it more challenging to review the actual changes here.